spidermon/contrib/actions/slack/__init__.py

```diff
@@ -85,27 +86,26 @@ def _get_user_id(self, username):
         return user["id"] if user else None
 
     def _get_users_info(self):
-        return dict(
-            [
-                (member["name"].lower(), member)
-                for member in self._api_call("users.list")["members"]
-            ]
-        )
-
-    def _api_call(self, method, **kwargs):
-        response = self._client.api_call(method, **kwargs)
-
-        has_errors = not response.get("ok")
-        if has_errors:
-            error_msg = response.get("error", {}).get("msg", "Slack API error")
-            logger.error(error_msg)
-
-        if isinstance(response, six.string_types):  # slackclient < v1.0
-            response = json.loads(response)
-        return response
+        try:
```

Comment:
**suggestion** Considering that we don't do anything special (like retrying the call or other action to recover from the exception) and just logging the exception text, I believe that removing the `try...except` is fine as the exceptions will be logged anyway. The same for all calls below.

Reply:
I'm adding the `try..except` blocks because of this warning: https://github.com/slackapi/python-slackclient/wiki/Migrating-to-2.x#error-handling-changes
Will Spidermon always log those exception even if we don't add them to `logger.error`? (sorry if it's an obvious question)

Reply:
Any exception raised will be sent to stderr, so we don't need to care about it.